src/spikeinterface_pipelines/pipeline.py

```diff
@@ -10,18 +10,19 @@
 from .postprocessing import postprocess, PostprocessingParams
 
 
-# TODO - WIP
 def run_pipeline(
     recording: si.BaseRecording,
-    scratch_folder: Path = Path("./scratch/"),
-    results_folder: Path = Path("./results/"),
-    job_kwargs: JobKwargs = JobKwargs(),
-    preprocessing_params: PreprocessingParams = PreprocessingParams(),
-    spikesorting_params: SpikeSortingParams = SpikeSortingParams(),
-    postprocessing_params: PostprocessingParams = PostprocessingParams(),
+    scratch_folder: Union[Path, str] = Path("./scratch/"),
```

Comment:
with python > 3.9 we can switch to `Path | str` annotation, which is way nicer!

Reply:
I agree! I got a problem with the spikeinterface sorter images though, which have 3.9 if I'm not mistaken